.github/workflows/lte-integ-test-magma-deb.yml

```diff
@@ -0,0 +1,64 @@
+# Copyright 2022 The Magma Authors.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: LTE integ test magma-deb
+
+on:
+  workflow_dispatch: null
+  workflow_run:
+    workflows:
+      - build-all
+    branches:
+      - master
+    types:
+      - completed
+
+jobs:
+  lte-integ-test-magma-deb:
+    if: github.repository_owner == 'magma' || github.event_name == 'workflow_dispatch'
```

Comment:
The previous correction from https://github.com/magma/magma/pull/13842/files#r963019350 is missing here again.

Reply:
Is done.